docs/rules/no-find-dom-node.md

```diff
@@ -1,6 +1,8 @@
-# Disallow usage of findDOMNode (react/no-find-dom-node)
+# Disallow usage of findDOMNode (`react/no-find-dom-node`)
 
-💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations): `all`, `recommended`.
+✅ This rule is enabled in the `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/README.md#shareable-configs).
```

Comment:
```suggestion
✅ This rule is enabled in the `recommended` [config](https://github.com/jsx-eslint/eslint-plugin-react/blob/HEAD/README.md#shareable-configs).
```
all these links should avoid hardcoding `master` in case we update it in the future; `HEAD` is futureproof.